src/litserve/api.py

```diff
@@ -113,76 +113,23 @@ def device(self):
     def device(self, value):
         self._device = value
 
-    def _sanitize(self, max_batch_size: int, spec: Optional[LitSpec]):
+    @property
+    def max_batch_size(self):
+        return self._max_batch_size
+
+    @max_batch_size.setter
+    def max_batch_size(self, value):
+        self._max_batch_size = value
```

Comment:
Why is the property when you use it as plain in and out?

Reply:
agree! removing it